tests/functional/test_loaded_model.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import grpc
+import pytest
+
+from tritonclient.grpc import service_pb2
+from tritonclient.grpc import service_pb2_grpc
+import tritonclient.grpc as grpcclient
+from model_config import MODEL_CONFIG
+PORT=8111
```

Comment:
is it possible to randomize value from available ports instead of hardcoded value?